microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -259,23 +324,41 @@ The main toolbar contains the following features:
    *  **Performance** tool.
    *  **Memory** tool.
    *  **Application** tool.
-   *  **Security** tool.
-   *  **Lighthouse** tool.
-   *  **CSS Overview** tool.
 
 * Icons:
-   *  **More tabs** (![More tabs icon](./overview-images/more-tabs-icon-light-theme.png)) button.
+   *  **Move Activity Bar to left** (![Move Activity Bar to left icon](./overview-images/move-activity-bar-to-left-icon.png)) button.
    *  **More Tools** (![More Tools icon](./overview-images/more-tools-icon-light-theme.png)) button.
-   *  **JavaScript errors counter** (![JavaScript errors counter icon](./overview-images/javascript-counter-icon-light-theme.png)) button.
-   *  **Issues counter** (![Issues counter icon](./overview-images/issues-counter-icon-light-theme.png)) button.
-   *  **Settings** (![Settings icon](./overview-images/settings-gear-icon-light-theme.png)) button.
-   *  **Send Feedback** (![Send Feedback icon](./overview-images/send-feedback-icon-light-theme.png)) button.
    *  **Customize and control DevTools** (![Customize icon](./overview-images/customize-devtools-icon-light-theme.png)) menu button.
+   *  **Help** (![Help icon](./overview-images/help-icon.png)) button.
    *  **Close** DevTools (![Close DevTools icon](./overview-images/close-devtools-icon-light-theme.png)) button.
 
 ![The main toolbar, with labels that identify its features](./overview-images/devtools-intro-menu-bar.png)
 
-The toolbar features are described below.
+The **Activity Bar** features are described below.
+
+
+<!-- ------------------------------ -->
+#### Pin and rearrange tools in the Activity Bar
+
+You can choose which tools appear in the **Activity Bar** by pinning or unpinning tools. This allows you to customize DevTools to your preferred workflow.
+
+Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. It will be shown by default every time DevTools is opened.
+
+![The More tools menu in Focus Mode, with the expanded list of all tools](./overview-images/more-tools.png)
+
+If there isn't enough space in the **Activity Bar** to display all pinned tools, some tools may overflow into the **More tools** menu:
```

Comment:
i think this is misleading.  it sounds automatic.  who is the actor?  i think this never happens unless user actively moves.  either say "automatically" or "you can move..."

Reply:
It happens automatically when the user keeps adding more tools to the activity bar. At some point, there isn't enough space to fit them all in the activity bar, and only a few get displayed there, while the rest gets displayed into the More Tools menu.